packages/ibm-products/src/components/APIKeyModal/APIKeyModal.tsx

```diff
@@ -289,6 +289,9 @@ export let APIKeyModal: React.FC<APIKeyModalProps> = forwardRef(
                       fileName={downloadFileName}
                       linkText={downloadLinkText}
                       fileType={downloadFileType}
+                      DownloadLinkLabel={
```

Comment:
Hi ,
Should we expose this as a optional prop , rather than hardcoding  and default to linkText ?